### PR TITLE
[GOBBLIN-1506] Disable CleanableMysqlDatasetStoreDatasetTest unit tests, already dropped from Continuous Ingetration.

### PR DESCRIPTION
**Pull request by phet**
Dear Gobblin maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

### JIRA
- [ ] My PR addresses the following [Gobblin JIRA](https://issues.apache.org/jira/browse/GOBBLIN/) issues and references them in the PR title.
    - https://issues.apache.org/jira/browse/GOBBLIN-1506

### Description
- [ ] Here are some details about my PR, including screenshots (if applicable):

Corroborated and consistently failing unit tests, which are already disabled for CI, are now likewise disabled for local build.

### Tests
- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

The change is itself isolated to unit tests.

### Commits
- [ ] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

**Review by Will-Lo — APPROVED** [review]

**Review by phet — COMMENTED** [review]
thinking on it, we may choose also whether to remove `groups = {"disabledOnCI"}` from the class-level annotation (line 53)

File: gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/Orchestrator.java [hunk 1]
```diff
@@ -288,13 +287,28 @@ public void orchestrate(Spec spec) throws Exception {
...
       }
 
       if (this.dagManager.isPresent()) {
-        //Send the dag to the DagManager.
-        this.dagManager.get().addDag(jobExecutionPlanDag, true, true);
+        try {
+          //Send the dag to the DagManager.
+          this.dagManager.get().addDag(jobExecutionPlanDag, true, true);
+        } catch (Exception ex) {
+          // pronounce failed before stack unwinds, to ensure flow not marooned in `COMPILED` state; (failure likely attributable to DB cnxn/failover)
```
Comment (autumnust) [inline gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/Orchestrator.java:299]: what does `cnxn` mean?

**Review by autumnust — CHANGES_REQUESTED** [review]
Thanks for contributing this PR! 

A general comment: Is the change in `Orchestrator` relevant ? It is usually better to maintain atomicity of a PR so orthogonal changes doesn't got checked in together. If they do, please consider changing the PR title to be more accurate.

File: gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/Orchestrator.java (hunk 1 above)
Comment (phet) [reply gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/Orchestrator.java:299]: "connection".  would you like me to update to write fully?

**Comment by phet** [comment]
closing, since replaced by:

https://github.com/apache/gobblin/pull/3355
https://github.com/apache/gobblin/pull/3357

**Pull request closed without merging**